CMakeLists.txt

```diff
@@ -75,3 +76,9 @@ add_custom_command(OUTPUT ${OUTPUT}
 add_custom_target(target ALL DEPENDS ${OUTPUT})
 
 install(TARGETS RUNTIME)
+
+
+include("${CMAKE_CURRENT_SOURCE_DIR}/cminx/cminx.cmake")
```

Comment:
I'd argue that it makes more sense for the `cminx.cmake` file to be in the `cmake` directory. If you move `cminx.cmake` into the `cmake` directory this becomes `include(cminx)`.

Reply:
Moved it to the cmake folder, also added doc generation for that folder which includes find_python_module